docs/docs-content/clusters/data-center/nutanix/create-manage-nutanix-cluster.md

```diff
@@ -0,0 +1,146 @@
+
+## Prerequisites
+
+- A Nutanix Prism Central cloud account.
+
+- A Private Cloud Gateway (PCG) deployed.
+
+- Minimum supported CAPI version v1.5.3. Refer to the Nutanix [Validated Integrations](https://opendocs.nutanix.com/capx/v1.2.x/validated_integrations/#validated-versions) compatibility matrix.
+
+- A cluster profile created for the Nutanix cloud. To learn how to create a profile, refer to  [Create Cluster Profiles](../../../profiles/cluster-profiles/create-cluster-profiles/).
+
+- A Cluster API (CAPI) image created for the Nutanix cloud platform. For guidance, refer to [Building CAPI Images for Nutanix Cloud Platform](https://image-builder.sigs.k8s.io/capi/providers/nutanix.html#building-capi-images-for-nutanix-cloud-platform-ncp).
+
+<!-- - The following YAML files obtained and created during cloud registration process:
+  - cloudClusterTemplate.yaml
+  - controlPlanePoolTemplate.yaml
+  - infrastructure-component.yaml
+  - workerPoolTemplate.yaml -->
+
+
+## Deploy a Nutanix Cluster
+
+Use the following steps to deploy a new Nutanix cluster.
+
+1. Log in to [Palette](https://console.spectrocloud.com/).
+
+2. From the left **Main Menu** select **Clusters**.
+
+3. Click on **Add New Cluster** and select **Deploy New Cluster** on the next page Palette displays. 
+
+4. Select **Nutanix** and click the **Start Nutanix Configuration** button.
+
+5. Fill out the following basic information, and click **Next** to continue.
+
+  | **Field** | **Description** |
+  |-----------|-----------------|
+  | **Cluster Name**| A custom name for the cluster. |
+  | **Description**| Use the description to provide context about the cluster.|
+  | **Tags**| Assign any desired cluster tags. Tags on a cluster are propagated to the Virtual Machines (VMs) deployed to the target environments.|
+  | **Cloud Account** | Select your Nutanix account from the **drop-down Menu**. If you have not yet added your account, select **Add New Account** and add your Nutanix account information. |
+
+6. Select the Nutanix cluster profile you created and click **Next**. Palette displays the cluster profile layers.
+
+7. Review the profile layers and customize parameters as desired in the YAML files that display when you select a layer. Click **Next** when you are done.
+
+8. In the Cluster configuration YAML file that Palette displays, edit the file to replace each occurrence of the variables within curly braces listed in the table with values that apply to your Nutanix cloud environment, and make any adjustments to configure your cluster. Click **Next** when you are done.
+
+  | **Variable** | **Description** |
+  |--------------|-----------------|
+  | `${CLUSTER_NAME}`| The name of the Nutanix workload cluster. |
+  | `${CONTROL_PLANE_ENDPOINT_IP}`| The host static IP address. |
+  | `${CLUSTER_ENDPOINT}`| The cluster IP address. |  
```

Comment:
How do we get the CLUSTER_ENDPOINT? Could we use macros with these variables?
Maybe we could try to improve the UX with a screenshot or a screen recording. What do you think?

Reply:
Discussed - they use Nutanix as their cluster endpoint.